images/Dockerfile.ubuntu-20-lts-arm-nvidia-jetson-agx-orin

```diff
@@ -28,6 +28,10 @@ ARG FRAMEWORK_VERSION=35.3
 
 RUN DEBIAN_FRONTEND=noninteractive apt-get install -y software-properties-common
 
+# nohang
+RUN add-apt-repository ppa:oibaf/test
+RUN apt-get install -y nohang
+
```

Comment:
This is not really needed since we don't build standard images for this flavor (https://github.com/kairos-io/kairos/pull/1689/files#diff-5cb44b260a8400267a5ff9938571e2022a5ef59730459941d173f5b5fffb389c).